### DISCUSSION TITLE
is there any way to protect projects from copying

### DISCUSSION THREAD

**Discussion by priyankarpal** [community/community#121433] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Hello maintainers and developers. I recently came across a post on X where someone had created a project, open-sourced it, and discovered that another person stole the code and launched a site with ads. Is there any way to protect a project from being copied without permission? 

Otherwise, open source is becoming risky.

Here is the actual project: https://www.animetrix.xyz

Copied one: https://animezia.com

X post: https://twitter.com/sh17va/status/1784507588417405340

github discussion of the copycat person: https://github.com/warlordsnet/animezia.com/issues/13

The person who copied: https://github.com/animemandir

**Comment by ZFPuhi** [comment, marked as answer]
Well there isn't a way to actually 'protect' it, but in this case you can simply contact the Hosting company with a DMC take down request as its a copy of your work.

In such DMC Take down requests you would need to provide them with details and proof that indeed its your project that got copied. Afterwards if you have registered the domain as an organization/company you can contact the Domain Registrar and provide them with the necessary information/documentation in order to actually block the domain itself also.

**Reply by Rifahad** [reply]
good #10053

**Reply by Rifahad** [reply]
git clone and add the link
